src/utils/timezone_names.js

```diff
@@ -0,0 +1,12 @@
+function getTimeZoneWithFormat(format) {
+	try {
+		const formatter = new Intl.DateTimeFormat('en-US', { timeZoneName: format });
+		return formatter.formatToParts(new Date()).find(part => {
+			return part.type === 'timeZoneName';
+		}) || '';
```

Comment:
```suggestion
		return formatter.formatToParts(new Date()).find(part => {
			return part.type === 'timeZoneName';
		}).value || '';
```

and remove the `.value` below. Right now, you would be trying to get `.value` from a string sometimes, which is not quite what we want.